PowerEditor/installer/nativeLang/english.xml

```diff
@@ -75,6 +75,7 @@
                     <Item id="41002" name="&amp;Open"/>
                     <Item id="41019" name="Explorer"/>
                     <Item id="41020" name="cmd"/>
+                    <Item id="41025" name="Folder as Workspace"/>
```

Comment:
We have already the same under `View` > `Folder as Workspace`

Reply:
So? It's not the same command.